docs/source/markdown/podman-create.1.md

```diff
@@ -706,6 +706,11 @@ Tune the host's OOM preferences for containers (accepts -1000 to 1000)
 #### **--os**=*OS*
 Override the OS, defaults to hosts, of the image to be pulled. For example, `windows`.
 
+#### **--personality**=*persona*
+
+Personality sets the execution domain via linux personality.
+Reference: https://man7.org/linux/man-pages/man2/personality.2.html
```

Comment:
It's more idiomatic to write `personality(2)` in man pages and to add that to the references at the very bottom of a manpage.

I suggest to remove this line and turn the one above into:
> Personality sets the execution domain via Linux personality(2).